CodeEdit/Features/Editor/Models/Editor.swift

```diff
@@ -106,22 +100,26 @@ final class Editor: ObservableObject, Identifiable {
 
     /// Closes a tab in the editor.
     /// This will also write any changes to the file on disk and will add the tab to the tab history.
```

Comment:
Which part in this function writes the changes to the file on disk?

Reply:
Isn't that handled by `guard canCloseTab(file: file) else { return }`?

Reply:
Yeah that's right, `canCloseTab` asks down to the `NSDocument` subclass to handle it.